taf/docker-compose-taf.yml

```diff
@@ -1004,10 +989,29 @@ services:
       DATABASES_PRIMARY_HOST: edgex-redis
       DATABASE_HOST: edgex-redis
       EDGEX_PROFILE: mqtt-export
```

Comment:
We have a conflict with service keys since this service is using the same profile as the standard app-mqtt-export. We need to use the `EDGEX_SERVICE_KEY` environment variable to override the service key for this instance. Something like `EDGEX_SERVICE_KEY: scalability-app-mqtt-export` ? Then this key needs to be added to the tokens list.

Reply:
I didn't find `EDGEX_SERVICE_KEY` env in the app-service-mqtt-export's configuration, where does it come from?
https://github.com/edgexfoundry/app-service-configurable/blob/master/res/mqtt-export/configuration.toml

Reply:
It isn't a config override. It is a specific override of the service key that the SDK implements.

Reply:
oh i see